### PR TITLE
Add type buttons for posting a new comment

### PR DESCRIPTION
**Pull request by n-wach**
This also includes the type in the new comment post request.

Screenshot: https://screenshot.googleplex.com/6s2eK762vxxkvxh

Unfortunately the buttons are hard coded into the HTML.  We can add these programmatically in the future, if we need to.

### CODE REVIEW

File: src/main/webapp/view/index.html [hunk 1]
```diff
@@ -48,9 +48,19 @@ <h2 id="header-text">ZoomTube</h2>
...
-        <button class="zt-btn btn btn-primary mb-2 float-right" type="button" onclick="postNewComment()">
-          Post Comment
-        </button>
+        <div class="btn-group-toggle" data-toggle="buttons" id="new-comment-types">
+          <label class="badge btn btn-outline-danger btn-sm float-left"
+            ><input type="radio" value="QUESTION" />Question</label
+          >
+          <label class="badge btn btn-outline-secondary btn-sm float-left active"
+            ><input type="radio" value="NOTE" />Note</label
+          >
```
Comment (n-wach) [inline src/main/webapp/view/index.html:57]: The linter keeps making it ugly here... not sure if I can do anything about it

File: src/main/webapp/view/discussion/discussion-area.js [hunk 2]
```diff
@@ -27,6 +26,10 @@ export const ELEMENT_DISCUSSION =
 export default class DiscussionArea {
   static #ELEMENT_POST_TEXTAREA = document.querySelector('#post-textarea');
   static #ELEMENT_TIMESTAMP_SPAN = document.querySelector('#timestamp-span');
+  static #ELEMENT_NEW_COMMENT_TYPES =
+      document.querySelector('#new-comment-types');
+  static #SELECTOR_SELECTED_TYPE = 'label.active > input';
```
Comment (jaysaleh) [inline src/main/webapp/view/discussion/discussion-area.js:31]: This is confusing can you add a comment here

File: src/main/webapp/view/discussion/discussion-area.js [hunk 3]
```diff
@@ -138,10 +141,16 @@ export default class DiscussionArea {
    * Posts the comment in the new comment area, and reloads the discussion.
    */
   postNewComment() {
+    /* eslint-disable indent */
+    const type = DiscussionArea.#ELEMENT_NEW_COMMENT_TYPES
```
Comment (jaysaleh) [inline src/main/webapp/view/discussion/discussion-area.js:145]: type is generic please make it more specific

Comment (n-wach) [reply src/main/webapp/view/discussion/discussion-area.js:145]: renamed to `commentType`

File: src/main/webapp/view/discussion/discussion-area.js (hunk 2 above)
Comment (n-wach) [reply src/main/webapp/view/discussion/discussion-area.js:35]: added one

File: src/main/webapp/view/index.html (hunk 1 above)
Comment (billie-cat) [reply src/main/webapp/view/index.html:57]: if you add a new line between type and value does that fix things?

Comment (n-wach) [reply src/main/webapp/view/index.html]: yeah that helps.  I added some other newlines and now it looks good

**Pull request merged**